src/java/org/apache/cassandra/net/MessagingService.java

```diff
@@ -312,13 +317,13 @@ public static MessagingService instance()
         OutboundConnections.scheduleUnusedConnectionMonitoring(this, ScheduledExecutors.scheduledTasks, 1L, TimeUnit.HOURS);
     }
 
-    public <T> org.apache.cassandra.utils.concurrent.Future<Message<T>> sendWithResult(Message message, InetAddressAndPort to)
+    public <REQ, RSP> org.apache.cassandra.utils.concurrent.Future<Message<RSP>> sendWithResult(Message<REQ> message, InetAddressAndPort to)
```

Comment:
nit: I guess this and a few other methods in this class should have `@Override` annotations now.